src/app/regular-giving/regular-giving.component.ts

```diff
@@ -230,6 +230,8 @@ export class RegularGivingComponent implements OnInit, AfterViewInit {
       let errorMessage = 'Form error: ';
       if (this.mandateForm.get('donationAmount')?.hasError('required')) {
         errorMessage += "Monthly donation amount is required";
```

Comment:
I don't think the above can ever actually happen, should have been validated in an earlier step. But not much harm leaving it here for now.